event-handlers/src/test/java/no/sikt/nva/nvi/events/persist/UpsertNviCandidateHandlerTest.java

```diff
@@ -129,7 +128,7 @@ void shouldSendMessageToDlqWhenUnexpectedErrorOccurs() {
 
     @Test
     void shouldSaveNewNviCandidateWithPendingInstitutionApprovalsWhenCandidateDoesNotExist() {
-        var evaluatedNviCandidate = randomEvaluatedNviCandidate().build();
+        var evaluatedNviCandidate = randomEvaluatedNviCandidate().withInstanceType("InvalidInstanceType").build();
```

Comment:
Thoughts @monaullah ? We added this to see if the test broke with an invalid instance type, but it didn't. Should we add validation or just keep accepting all strings?

Reply:
Yes, we should validate InstanceType, but maybe in a new PR?
For now, I would remove `.withInstanceType("InvalidInstanceType")` from this test, and create a subtask in jira with to follow this up.